build-aux/snap/snapcraft.yaml

```diff
@@ -203,10 +203,23 @@ parts:
       craftctl default
       # set version, this needs dpkg-parsechangelog (from dpkg-dev) and git
       VERSION="$(./mkversion.sh --output-only)"
-      if [ -f fips-build ] ; then
-          echo "-- appending FIPS tag to version $VERSION"
-          VERSION="$VERSION-fips"
+      if [ "${VERSION%/+fips/}" != "$VERSION" ] ; then
```

Comment:
Did you mean `"${VERSION%+fips}"` or `"${VERSION/%+fips/}"`?

Reply:
ah yes, too many changes going back and forth, `/+fips/` let me fix that

Reply:
it's just a simplification, debian/rules checks simply checks for +fips too, but we'll only ever append +fips as a suffix int eh changelog